content/manuals/docker-hub/quickstart.md

```diff
@@ -1,148 +1,221 @@
-  organizations, repositories, groups, teams
+keywords: Docker Hub, push image, pull image, repositories
 title: Docker Hub quickstart
 linkTitle: Quickstart
 weight: 10
 ---
 
-The following section contains step-by-step instructions on how to get started with Docker Hub.
+Docker Hub provides a vast library of pre-built images and resources,
+accelerating development workflows and reducing setup time. You can build upon
+pre-built images from Docker Hub and then use repositories to share and
+distribute your own images with your team or millions of other developers.
 
-### Step 1: Sign up for a free Docker account
+This guide shows you how to find and run a pre-built image. It then walks you
+through creating a custom image and sharing it through Docker Hub.
 
-Start by creating a [Docker ID](https://hub.docker.com/signup).
+## Prerequisites
 
-A [Docker ID](/accounts/create-account/) grants you access to Docker Hub repositories and lets you explore available images from the community and verified publishers. You also need a Docker ID to share images on Docker Hub.
+- [Download and install Docker](../../get-started/get-docker.md)
+- [Create a Docker account](https://app.docker.com/signup)
 
-> [!TIP]
->
-> Explore [Docker's core subscriptions](https://www.docker.com/pricing/) to see what else Docker can offer you. 
+## Step 1: Find an image in Docker Hub's library
 
-### Step 2: Create your first repository
+You can search for content in Docker Hub itself, in the Docker Desktop
+Dashboard, or by using the `docker search` CLI command. Searching on Docker Hub
+itself offers the most options to explore content.
 
-To create a repository:
+To search or browse for content on Docker Hub:
 
-1. Sign in to [Docker Hub](https://hub.docker.com).
-2. On the Repositories page, select **Create repository**.
-3. Name it **&lt;your-username&gt;/my-private-repo**.
-4. Set the visibility to **Private**.
-5. Select **Create**.
+1. Navigate to the [Docker Hub Explore page](https://hub.docker.com/explore).
 
-You've created your first repository.
+   On the Explore page, you can browse by catalog or category, or use the search
+   to quickly find content.
 
-### Step 3: Download and install Docker Desktop
+2. Under **Categories**, select **Web servers**.
 
-You need to download Docker Desktop to build, push, and pull container images.
+   After the results are displayed, you can further filter the results using the
+   filters on the left side of the page.
 
-1. Download and install [Docker Desktop](/manuals/desktop/_index.md).
+3. In the filters, select **Docker Official Image**.
 
-2. Sign in to Docker Desktop using the Docker ID you created in step one.
+   Filtering by Trusted Content ensures that you see only high-quality, secure
+   images, curated by Docker and verified publishing partners.
 
-### Step 4: Pull and run a container image from Docker Hub
+4. In the results, select the Nginx image.
```

Comment:
more of a question/nit, should **Ngix** or **Ngix image** be bold if they are clicking on it?

Reply:
Yea, I was trying to appease vale. I'll bold it and make it literal UI text. The literal UI text is `nginx`, but vale nags about the capitalization.